recipes/configs/llama3/70B_full.yaml

```diff
@@ -17,7 +17,7 @@
 #
 # !!!!!!!!!!!!!
 # !!!!!!!!!!!!!
-# ATTENTION: It will only work with pytorch>=2.5 (nightlies). For other pytorch versions, it will OOM, even on 8xA100.
```

Comment:
I think we can remove this all together. We make no guarantees to really support any versions of PyTorch with the exception of the latest PyTorch and nightlies.

Reply:
Ok